### PR TITLE
DCS-808 bvl bugfix room avaiability on amend appointment

### PR DESCRIPTION
**Pull request by GurnankCheema**
…appointment details are ignored in the 'unavailable' data set returned.

### CODE REVIEW

File: backend/routes/createBooking/selectRoomsController.ts [hunk 1]
```diff
@@ -110,7 +110,7 @@ export default function selectCourtAppointmentRoomsFactory({
   notifyApi,
 }: Services): Handlers {
   const index = async (req, res) => {
-    const { offenderNo, agencyId } = req.params
+    const { offenderNo, agencyId, bookingId: appointmentBookingId } = req.params
```
Comment (andrewrlee) [inline backend/routes/createBooking/selectRoomsController.ts:113]: If this existed , it wouldn't represent the video booking id as we haven't created it yet.

(We wouldn't refer to it as appoinment - a booking is made up of 3 appointments)

File: backend/services/availabilityCheckService.test.ts [hunk 2]
```diff
@@ -29,9 +29,11 @@ describe('AvailabilityCheckService', () => {
     description: `Room-${locationId}`,
   })
 
+  const bookingId = 123
```
Comment (andrewrlee) [inline backend/services/availabilityCheckService.test.ts:32]: This should be referred to as `videoBookingId`

Need a test for when we aren't passing it through?

File: backend/routes/createBooking/selectRoomsController.test.ts [hunk 3]
```diff
@@ -24,7 +24,7 @@ describe('Select court appointment rooms', () => {
 
   const req = {
     originalUrl: 'http://localhost',
-    params: { agencyId: 'MDI', offenderNo: 'A12345' },
+    params: { agencyId: 'MDI', offenderNo: 'A12345', bookingId: '123' },
```
[the remark this answers is not in the document]
Comment (GurnankCheema) [reply backend/routes/createBooking/selectRoomsController.test.ts:27]: done

File: backend/routes/createBooking/selectRoomsController.ts (hunk 1 above)
Comment (GurnankCheema) [reply backend/routes/createBooking/selectRoomsController.ts:113]: done

File: backend/services/availabilityCheckService.test.ts (hunk 2 above)
Comment (GurnankCheema) [reply backend/services/availabilityCheckService.test.ts:32]: done

**Comment by GurnankCheema** [comment]
this work has now been done in another PR

**Pull request closed without merging**